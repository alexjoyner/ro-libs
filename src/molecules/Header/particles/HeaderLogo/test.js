import React from 'react';
import { mount } from 'enzyme';
import { HeaderLogo } from './';

describe('HeaderLogo component', () => {
  let component;
  beforeEach(() => {
    component = mount(<HeaderLogo logoText="Testing" />);
  });
  it('Should render without exploding', () => {
    expect(component).toBeDefined();
  });
  it('Should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
  it('should default to white font color', () => {
    expect(component).toHaveStyleRule('color', 'white');
  });
  it('should apply a passed font color', () => {
    const colorTest = mount(<HeaderLogo logoText="Testing" fontColor="blue" />);
    expect(colorTest).toHaveStyleRule('color', 'blue');
  });
});
