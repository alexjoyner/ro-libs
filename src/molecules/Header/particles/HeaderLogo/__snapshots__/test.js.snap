// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderLogo component Should match snapshot 1`] = `
.c0 {
  padding-left: 20px;
  color: white;
}

<Styled(Logo)
  logoText="Testing"
>
  <Logo
    className="c0"
    logoText="Testing"
  >
    <styled.span
      className="c0"
      logoText="Testing"
    >
      <span
        className="c0 "
      >
        Testing
      </span>
    </styled.span>
  </Logo>
</Styled(Logo)>
`;
